handler/src/main/java/io/netty/handler/ssl/OpenSslCertificateException.java

```diff
@@ -24,6 +27,286 @@
+    public static final int X509_V_ERR_UNABLE_TO_VERIFY_LEAF_SIGNATURE;
+    public static final int X509_V_ERR_CERT_CHAIN_TOO_LONG;
+    public static final int X509_V_ERR_CERT_REVOKED;
+    public static final int X509_V_ERR_INVALID_CA;
+    public static final int X509_V_ERR_PATH_LENGTH_EXCEEDED;
+    public static final int X509_V_ERR_INVALID_PURPOSE;
+    public static final int X509_V_ERR_CERT_UNTRUSTED;
+    public static final int X509_V_ERR_CERT_REJECTED;
+    public static final int X509_V_ERR_SUBJECT_ISSUER_MISMATCH;
+    public static final int X509_V_ERR_AKID_SKID_MISMATCH;
+    public static final int X509_V_ERR_AKID_ISSUER_SERIAL_MISMATCH;
+    public static final int X509_V_ERR_KEYUSAGE_NO_CERTSIGN;
+    public static final int X509_V_ERR_UNABLE_TO_GET_CRL_ISSUER;
+    public static final int X509_V_ERR_UNHANDLED_CRITICAL_EXTENSION;
+    public static final int X509_V_ERR_KEYUSAGE_NO_CRL_SIGN;
+    public static final int X509_V_ERR_UNHANDLED_CRITICAL_CRL_EXTENSION;
+    public static final int X509_V_ERR_INVALID_NON_CA;
+    public static final int X509_V_ERR_PROXY_PATH_LENGTH_EXCEEDED;
+    public static final int X509_V_ERR_KEYUSAGE_NO_DIGITAL_SIGNATURE;
+    public static final int X509_V_ERR_PROXY_CERTIFICATES_NOT_ALLOWED;
+    public static final int X509_V_ERR_INVALID_EXTENSION;
+    public static final int X509_V_ERR_INVALID_POLICY_EXTENSION;
+    public static final int X509_V_ERR_NO_EXPLICIT_POLICY;
+    public static final int X509_V_ERR_DIFFERENT_CRL_SCOPE;
+    public static final int X509_V_ERR_UNSUPPORTED_EXTENSION_FEATURE;
+    public static final int X509_V_ERR_UNNESTED_RESOURCE;
+    public static final int X509_V_ERR_PERMITTED_VIOLATION;
+    public static final int X509_V_ERR_EXCLUDED_VIOLATION;
+    public static final int X509_V_ERR_SUBTREE_MINMAX;
+    public static final int X509_V_ERR_APPLICATION_VERIFICATION;
+    public static final int X509_V_ERR_UNSUPPORTED_CONSTRAINT_TYPE;
+    public static final int X509_V_ERR_UNSUPPORTED_CONSTRAINT_SYNTAX;
+    public static final int X509_V_ERR_UNSUPPORTED_NAME_SYNTAX;
+    public static final int X509_V_ERR_CRL_PATH_VALIDATION_ERROR;
+    public static final int X509_V_ERR_PATH_LOOP;
+    public static final int X509_V_ERR_SUITE_B_INVALID_VERSION;
+    public static final int X509_V_ERR_SUITE_B_INVALID_ALGORITHM;
+    public static final int X509_V_ERR_SUITE_B_INVALID_CURVE;
+    public static final int X509_V_ERR_SUITE_B_INVALID_SIGNATURE_ALGORITHM;
+    public static final int X509_V_ERR_SUITE_B_LOS_NOT_ALLOWED;
+    public static final int X509_V_ERR_SUITE_B_CANNOT_SIGN_P_384_WITH_P_256;
+    public static final int X509_V_ERR_HOSTNAME_MISMATCH;
+    public static final int X509_V_ERR_EMAIL_MISMATCH;
+    public static final int X509_V_ERR_IP_ADDRESS_MISMATCH;
+    public static final int X509_V_ERR_DANE_NO_MATCH;
+    private static final IntObjectMap<Boolean> ERROR_MAP;
+
+    static {
+        OpenSsl.ensureAvailability();
```

Comment:
Still I not get why you think we should remove the init of OpenSSL in the static block as added here.... also be aware that if you use an übt that is not supported by Tonstufe it will throw an exception when init this class